README.md

```diff
@@ -13,9 +13,9 @@
 
 ## About
 
-Icinga DB serves as a synchronisation daemon between Icinga 2 (Redis) and Icinga Web 2 (MySQL). It synchronises configuration, state and history of an Icinga 2 environment using checksums.
+Icinga DB serves as a synchronisation daemon between Icinga 2 (Redis) and Icinga Web 2 (RDBMS). It synchronises configuration, state and history of an Icinga 2 environment using checksums.
```

Comment:
I would write `database` instead of `RDBMS` since we're speaking of `database` later on anyways. Also it makes sense to clarify that `database` means MySQL, MariaDB or PostgreSQL.